### ISSUE TITLE
Consider creating a README.md to help others help you

### ISSUE THREAD

**Issue by Bernd-L** [moesalih/spacex.moesalih.com#2]
Creating a `README.md` file helps others understand your project better.

You could discuss how your application is built, and how others can help you do so.

Go to <https://github.com/moesalih/spacex.moesalih.com/new/master?readme=1> and get started.

If you're not familiar with Markdown, you can learn more here:
<https://guides.github.com/features/mastering-markdown/>

**Comment by solomonshalom** [comment]
I agree, this project is very amazing and has the potential for a lot of people to contribute to it. If you can include a readme, that would be so better.
